### DISCUSSION TITLE
"Shared with users" - not showing Groups

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14853] [How to]
Created new module, and "Shared with users" field dont show Groups but only users. Is it something I need to fix somewhere manually? Was searching but no luck

Best

**Comment by vovpff** [comment]
You need to tune your groups before.

**Comment by waran70** [comment]
Fount it.... however it happens I have 3400 groups. Gonna look at dbase better

Thank you

**Reply by mariuszkrzaczkowski** [reply]
3400!! Wow 
why so much?

**Reply by waran70** [reply]
xperimenting, but it may go live. Been working on many crazy scenarios O.o

**Comment by s2b-git** [comment]
Thanks all. Solved
